automates/text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala

```diff
@@ -536,9 +543,35 @@ object ExtractAndAlign {
         val offsets = mention.tokenInterval.toString()
         val textVar = mention.arguments(VARIABLE).head.text
         val definition = mention.arguments(DEFINITION).head.text
+        val charBegin = mention.startOffset
+        val charEnd = mention.endOffset
+        val continuousMenSpanJson = if (textInputFormat == "cosmos") {
+          val attAsJson = mention.attachments.head.asInstanceOf[MentionLocationAttachment].toUJson.obj
+          val page = attAsJson("pageNum").num.toInt
+          val block = attAsJson("blockIdx").num.toInt
+
+          // todo: this is for mentions that came from one block
+          // mentions that come from separate cosmos blocks will require additional processing and can have two location spans
+          ujson.Obj(
+            "page" -> page,
+            "block" -> block,
+            "span" -> ujson.Obj(
+              "char_begin" -> charBegin,
+              "char_end" -> charEnd
+                      )
+                    )
+        } else {
+          ujson.Obj(
+            "page" -> "N/A",
```

Comment:
@maxaalexeeva please use `null` as opposed to the string _N/A_. That will produce the null object in JSON that will be read as `None` in Python. We are using `None` to be able to tell when a key does not have any object across all of our JSON specs in AutoMATES.

Additionally, I think you are outputting the string "None" in some locations. Can you please change those to be `null` as well?

Reply:
@pauldhein good points. I have addressed the null issue to the extent that we do not output any none or NA strings in the align endpoint. There are still some "None"s here and there in the code, but none of them make it into the output. I will try to get rid of those as I adjust the endpoints based on the meeting Fri, Feb 18, but I'd like to not spend more time on this change right now since some of that code might be gone anyway. Does that work?

Reply:
@maxaalexeeva as long as you addressed the N/As here then we can move forward with the merge. I agree that you shouldn't spend any more time tracking down additional N/As or nones until we know what code will be staying in the repo.